index.html

```diff
@@ -1944,16 +1887,12 @@ <h2>
           </li>
           <li>In a browser that supports Payment Handler API, when a merchant
           creates a PaymentRequest object with URL-based payment method
-          identifiers, <a>CanMakePaymentEvent</a> will fire in registered
-          payment handlers from a finite number of origins: the origins of the
+          identifiers, a <a>CanMakePaymentEvent</a> will fire in registered
+          payment handlers from a finite set of origins: the origins of the
           payment method manifests and their <a>supported origins</a>. This
-          means that a registered payment handler will know that a user has
-          visited a website before the user has selected that payment handler
-          to complete a transaction. This behavior is similar to the status quo
-          where a merchant embeds a third-party iframe in a checkout page.
-          However, because user agents enable users to disable the
-          <a>CanMakePaymentEvent</a> and users can choose to uninstall payment
-          handlers, this approach improves upon the iframe status quo.
+          event is fired before the user has selected that payment handler,
+          but it contains no information about the triggering origin (i.e.,
+          the merchant website) and so cannot be used to track users.
```

Comment:
> " and so cannot be used to track users"

Maybe remove this, because timing attacks still exist?

Reply:
I don't think there's value in being coy about these things. If we think there is a timing attack, we should document that instead. I've done that in the latest patch-set, WDYT?